### ISSUE TITLE
Sway crashes on Alpine Linux in QEMU with QXL graphics

### ISSUE THREAD

**Issue by fourstepper** [swaywm/sway#5839]
### Please fill out the following:
- **Sway Version:**
In the first screenshot in the description 

OS: Alpine Linux 3.12 in QEMU with QXL Graphics on a Fedora 33 host (running Intel GPU)

- **Debug Log:**
In the second screenshot

- **Configuration File:**
This is the default configuration

- **Description:**
  - The steps you took in plain English to reproduce the problem.

Start Sway

Sway refuses to start with Failed to open any DRM device (ss 1 & 2)

![image](https://user-images.githubusercontent.com/49439044/100328035-591f7600-2fcc-11eb-9c49-9692aa221294.png)
![image](https://user-images.githubusercontent.com/49439044/100328202-8ec45f00-2fcc-11eb-98df-e841cde2611b.png)

Perhaps there is something I am missing, if you could point that out to me that would be great as well - I will be happy to further provide any needed information

**Comment by emersion** [comment]
xf86 user-space display drivers are unnecessary. Everything is in the kernel.

Cat you `ls -l /dev/dri`?

**Comment by fourstepper** [comment]
That reports no such file or directory

**Comment by emersion** [comment]
That means the kernel driver for you GPU hasn't been loaded properly. Check your dmesg and ask help from Alpine folks.

**Comment by fourstepper** [comment]
Thanks for the help - I will try to debug this or turn to the Alpine community

**Comment by xerz-one** [comment]
If it's useful to anyone with this issue: make sure to remove `nomodeset` from the boot parameters! You can modify that permanently from `/boot/extlinux.conf` and `/etc/update-extlinux.conf`

**Comment by fourstepper** [comment]
This has not been a problem with sway, closing. Thank you for your time @emersion
